### DISCUSSION TITLE
Add a new V380 clone model?

### DISCUSSION THREAD

**Discussion by aleguedes1983** [Arkady23/V380-Web#18] [Q&A]
Hi @Arkady23,
Thank you for dedicating some part of your time whit this project. Definitely you are helping a lot of people here..

Do you mind to help me to create my installer? Not sure if I was able to capture the data, follow it bellow:

param=eyJhcHBfaWQiOjIsIm1vZGUiOiJsaWdodCIsImFkX3ZlciI6MywidmVyX2lkIjoxOTc5LCJ0eXBlIjo0MCwibGFuZ3VhZ2UiOiJlbiIsInNpemUiOjEwfQ==

![image](https://user-images.githubusercontent.com/21367963/138002462-8effe3b1-6045-4100-ac14-46b34bd04b1c.png)

Thank you very much for your help.

**Comment by Arkady23** [comment]
There is no HwName here. The Chinese have changed something. Now the update is different on new cameras. We need to figure it out. I don't know how yet and I don't have such a new camera.

**Reply by aleguedes1983** [reply]
That is fine.  I will try to connect on the Serial console, and try to discover something. I will keep you posted!
Thank you.
